add small block about redis configuration (#8073)
magento/devdocs: src/guides/v2.3/config-guide/redis/config-redis.md

*  Redis supports on-disk save and master/slave replication.

{:.bs-callout-info}

> Starting in Magento 2.0.6, you can use either Redis or [memcached]({{ page.baseurl }}/config-guide/memcache/memcache.html) for session storage. Earlier issues

Reviewer: ```suggestion
Starting from Magento 2.0.6, you can use either Redis or [memcached]({{ page.baseurl }}/config-guide/memcache/memcache.html) for session storage. Earlier issues
```

To optimize the Redis instance for your requirements, you get best results by using a dedicated instance for each session, Magento cache and FPC.

For sessions, we recommend that you enable persistence to copy Redis data to disk using either of the following persistence options: regular Redis Database Backup (RDB) snapshots, or Append Only File (AOF) persistence logs.

*  RDB (Redis Database Backup) snapshots store the complete database in a dump file after a given time, when a minimum number of keys have changed since the last save. Use the `save` setting inside the `redis.conf` file to configure this setting.

*  AOF (Append Only File) stores each write operation sent to Redis in a journal file. Redis reads this file on restart only and uses it to restore the original dataset.

You can also enable both the RDB and AOF options at the same time. For additional details including the advantages and disadvantages of the persistence options, see the [Redis Persistence documentation](https://redis.io/topics/persistence).

For the cache instance, set up the instance so that it is large enough to store your entire Magento cache.

> Size requirements depend on different factors like the number of products and store views, but the size requirement for the the file system cache gives you the order of magnitude. Persistence is not required for the cache instance because the Magento cache can be restored. See also [Redis cache guide](https://redis.io/topics/lru-cache).

Reviewer: @jonashrem  Can you provide an example for the cache instance size?

Author: @meker12 I just made some changes to the sentence. Is this clear now?

Author: >  Size requirements depend on different factors like the number of products and store views. When you have your Magento Cache previously put into your file system, you can check out the size of the cache folder for a good starting point to set up your Redis instance. For example, when you have a 5 GB 'var/cache' folder your Redis instance should have a similar size available. Persistence is not required for the cache instance because the Magento cache can be restored. See also [Redis cache guide](https://redis.io/topics/lru-cache).

*  [digitalocean](https://www.digitalocean.com/community/tutorials/how-to-install-and-use-redis)
*  [Redis documentation page](http://redis.io/documentation)

## Set up redis configuration {#config-redis-setup}

Depending on your installation, you can usually find your Redis configuration at /etc/redis/redis.conf or /etc/redis/`port`.conf

To optimize your Redis instance to your needs, you get best results when using a dedicated instance for each sessions, Magento cache and FPC.

> For sessions, it's recommended to enable persistence. This can either be done by regular RDB snapshots or by using the AOF persistence logs.

Author: Thanks for the Feedback. I just added the full definition.

You can get the details advantages and disadvantages on the [Redis Persistence documentation](https://redis.io/topics/persistence).

> For the cache instance, you should make sure, the instance is set up big enough to store your whole Magento cache in it.

Reviewer 2: ```suggestion
For the cache instance, you should make sure the instance is set up to be large enough to store your whole Magento cache.
```

*  [digitalocean](https://www.digitalocean.com/community/tutorials/how-to-install-and-use-redis)
*  [Redis documentation page](http://redis.io/documentation)

## Set up redis configuration {#config-redis-setup}

> Depending on your installation, you can usually find your Redis configuration at /etc/redis/redis.conf or /etc/redis/`port`.conf

Author: it's not literally /etc/redis/port.conf but a file with the port number in it like /etc/redis/6379.conf so the new formatting might be confusing. @meker12

To optimize your Redis instance to your needs, you get best results when using a dedicated instance for each sessions, Magento cache and FPC.

For sessions, it's recommended to enable persistence. This can either be done by regular RDB snapshots or by using the AOF persistence logs.
You can get details of the advantages and disadvantages of RDB and AOF on the [Redis Persistence documentation](https://redis.io/topics/persistence).

RDB (Redis Database File) snapshots store the complete database in a dump file after a given time, when a minimum number of keys have changed since the last save.
This can be configured with the `save` setting inside `redis.conf`.

> AOF (Append Only File) stores each write operation send to redis in a journal file. This file ẃill be read by Redis on restart to restore the the original dataset.

Reviewer 2: ```suggestion
AOF (Append Only File) stores each write operation sent to Redis in a journal file. This file ẃill be read by Redis on restart to restore the original dataset.
```

> It's possible, to enable both RDB ands AOF the same time.

Reviewer 2: ```suggestion
It's possible to enable both RDB and AOF at the same time.
```

> For the cache instance, you should make sure the instance is set up to be large enough to store your whole Magento cache.
> The required size depends on different factors (like number of products and store views) but the required size of the file system cache gives you the order of
> magnitude. Persistence is not required here as the Magento cache can be restored. See also [Redis cache guide](https://redis.io/topics/lru-cache).
> 
> For performance tuning you can also enable these settings for asynchronous deletion. This will not change the behavior of Redis (see also

Reviewer: Suggested revision of above paragraphs-- also, something missing `(see also..?).   

The size requirement sentence is a bit hard to understand. Not sure what you mean by "file system cache` gives you the order of magnitude.  A specific example might be helpful if it's possible to provide one.

```suggestion
For the cache instance, set up the instance so that it is large enough to store your entire Magento cache.
Size requirements depend on different factors like the number of products and store views, but the size requirement for the the file system cache gives you the order of magnitude. Persistence is not required here as the Magento cache can be restored. See also [Redis cache guide](https://redis.io/topics/lru-cache).

For performance tuning, you can also enable the following settings for asynchronous deletion. These settings do not change the behavior of Redis.
```

Author: > Suggested revision of above paragraphs-- also, something missing `(see also..?).
> 
> The size requirement sentence is a bit hard to understand. Not sure what you mean by "file system cache` gives you the order of magnitude. A specific example might be helpful if it's possible to provide one.

I mean that when you have a 20 GB Magento file system cache it won't fit into a 128 MB Redis instance

[redis news](http://antirez.com/news/93).

```ini
lazyfree-lazy-eviction yes
lazyfree-lazy-expire yes
lazyfree-lazy-server-del yes
replica-lazy-flush yes
```

> From redis 6.x onwards, you can also set

Reviewer: ```suggestion
On Redis 6.x and later, you can also add the following value:
```

You can also enable both the RDB and AOF options at the same time. For additional details including the advantages and disadvantages of the persistence options, see the [Redis Persistence documentation](https://redis.io/topics/persistence).

For the cache instance, set up the instance so that it is large enough to store your entire Magento cache.

> Size requirements depend on different factors like the number of products and store views. When you have your Magento Cache previously put into your file system, you can check out the size of the cache folder for a good starting point to set up your Redis instance. For example, when you have a 5 GB 'var/cache' folder your Redis instance should have a similar size available. Persistence is not required for the cache instance because the Magento cache can be restored. See also [Redis cache guide](https://redis.io/topics/lru-cache).

Reviewer: Much better with specific example.

```suggestion
Size requirements depend on different factors like the number of products and store views. As a starting point, you can use the size of the cache folder on your file system.  For example, if the `var/cache` folder on your file system is  5 GB, set up your Redis instance with at least 5 GB to start. Persistence is not required for the cache instance because the Magento cache can be restored. See also [Redis cache guide](https://redis.io/topics/lru-cache).
```
